docs/docs/identity-providers.md

```diff
@@ -10,15 +10,17 @@ description: >-
 
 This article describes how to configure Pomerium to use a third-party identity service for single-sign-on.
 
-There are a few configuration steps required for identity provider integration. Most providers support [OpenID Connect] which provides a standardized interface for IdentityProvider. In this guide we'll cover how to do the following for each identity provider:
+There are a few configuration steps required for identity provider integration. Most providers support [OpenID Connect] which provides a standardized identity and authentication interface.
 
-1. Establish a **Redirect URL** with the identity provider which is called after IdentityProvider.
+In this guide we'll cover how to do the following for each identity provider:
+
+1. Set a **[Redirect URL]** pointing back to Pomerium.
 2. Generate a **[Client ID]** and **[Client Secret]**.
 3. Configure Pomerium to use the **[Client ID]** and **[Client Secret]** keys.
 
 ## Azure Active Directory
 
-If you plan on allowing users to log in using a Microsoft Azure Active Directory account, either from your company or from external directories, you must register your application through the Microsoft Azure portal. If you don't have a Microsoft Azure account, you can [signup](https://azure.microsoft.com/en-us/free) for free.
+If you plan on allowing users to log in using a Microsoft Azure Active Directory account, either from your company or from external directories, you must register your application through the Microsoft Azure portal. If you don't have a Microsoft Azure account, you can [signup](https://azure.c.com/en-us/free) for free.
```

Comment:
@desimone I think you mistakenly made a typo here :)

Reply:
Most certainly did. Thank you!